comms/src/outbound_message_service/outbound_message_service.rs

```diff
@@ -85,49 +80,44 @@ pub enum OutboundError {
 /// Handler functions use the OutboundMessageService to send messages to peers. The OutboundMessage service will receive
 /// messages from handlers, apply a broadcasting strategy, encrypted and serialized the messages into OutboundMessages
 /// and write them to the outbound message pool.
-pub struct OutboundMessageService<PubKey, SecKey, DS> {
```

Comment:
Was removing the generics the right thing to do? It does make it easier having them hard coded in types.rs. Also, every module that needs the current nodes identity could be initialised with an Arc reference to the single copy of it.

Reply:
It made things a ton easier and is related to the static `NodeIdentity::global()` which could not use generics